### ISSUE TITLE
Unread/downloaded count in badges doesn't get updated imediately.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#8234]
### Steps to reproduce

1. Have unread / download count badges updated in library.
2. Read a chapter or delete a chapter.
3. Go back to the library screen.

### Expected behavior

Unread/downloaded count in the badges get updated immediately.

### Actual behavior

Unread/downloaded count in the badges doesn't get updated immediately and requires a tab change.

### Tachiyomi version

Preview r5062

### Android version

Android 11

### Device

Redmi Note 9 Pro Max

### Other details

This is probably caused same bug which causes #8222 but making a separate issue in case it isn't. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Seems fine after #8240 at least.
